examples/simple/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+#
+# gpt-2
```

Comment:
:)

i have seen "gpt-2" so often as a copy paste leftover in different projects now, it basically a meme.